pkgs/development/cuda-modules/generic-builders/manifest.nix

```diff
@@ -296,11 +301,14 @@ backendStdenv.mkDerivation (
 
     # For each output, create a symlink to it in the out output.
     # NOTE: We must recreate the out output here, because the setup hook will have deleted it if it was empty.
+    # TODO: Previously we used `meta.getExe lndir` to get the path to lndir, but that doesn't work under
+    # cross-compilation -- whatever machinery Nixpkgs uses to get a version built for hostPlatform (so it can run
+    # during the build) doesn't extend to `meta.getExe`.
```

Comment:
Too long and informal, but also I imagine that `getExe ((p: p.__spliced.pkgsBuildHost or p) lndir)` would have the same effect. Naturally `nativeBuildInputs` is the preferred solution

Reply:
Resolved in latest; reworded and referenced the tracking issue you made for getting rid of the fat output.